### PR TITLE
Add atand function

### PR DESCRIPTION
**Pull request by AdamRJensen**
- [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

pvlib has convenience functions for calculating sine, cosine, and tangent in degrees. Similarly, it has functions for arcsine and arccosine but NOT arctangent, which seems inconsistent.

I couldn't find anywhere these functions are tested, but maybe they just aren't?

### CODE REVIEW

File: pvlib/tools.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 def cosd(angle):
     """
-    Cosine with angle input in degrees
+    Cosine with angle input in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:14]:
```suggestion
    Trigonometric cosine with angle input in degrees.
```

File: pvlib/tools.py [hunk 2]
```diff
@@ -23,14 +23,13 @@ def cosd(angle):
...
     """
-
     res = np.cos(np.radians(angle))
     return res
 
 
 def sind(angle):
     """
-    Sine with angle input in degrees
+    Sine with angle input in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:32]:
```suggestion
    Trigonometric sine with angle input in degrees.
```

File: pvlib/tools.py [hunk 3]
```diff
@@ -42,14 +41,13 @@ def sind(angle):
...
     """
-
     res = np.sin(np.radians(angle))
     return res
 
 
 def tand(angle):
     """
-    Tan with angle input in degrees
+    Tangent with angle input in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:50]:
```suggestion
    Trigonometric tangent with angle input in degrees.
```

File: pvlib/tools.py [hunk 4]
```diff
@@ -61,14 +59,13 @@ def tand(angle):
...
     """
-
     res = np.tan(np.radians(angle))
     return res
 
 
 def asind(number):
     """
-    Inverse Sine returning an angle in degrees
+    Inverse Sine returning an angle in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:68]:
```suggestion
    Trigonometric inverse Sine returning an angle in degrees.
```

File: pvlib/tools.py [hunk 5]
```diff
@@ -80,14 +77,13 @@ def asind(number):
...
     """
-
     res = np.degrees(np.arcsin(number))
     return res
 
 
 def acosd(number):
     """
-    Inverse Cosine returning an angle in degrees
+    Inverse Cosine returning an angle in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:86]:
```suggestion
    Trigonometric inverse Cosine returning an angle in degrees.
```

File: pvlib/tools.py [hunk 6]
```diff
@@ -99,11 +95,28 @@ def acosd(number):
...
         arccos result
     """
-
     res = np.degrees(np.arccos(number))
     return res
 
 
+def atand(number):
+    """
+    Inverse Tangent returning an angle in degrees.
```
Comment (adriesse) [inline pvlib/tools.py:104]:
```suggestion
    Trigonometric inverse Tangent returning an angle in degrees.
```

**Comment by AdamRJensen** [comment]
I will say that I wish we would have named inverse sine ``arcsind`` and not ``asind`` to stay consistent with the Numpy naming which calls the radian equivalent (which we wrap) for ``np.arcsin``. @kandersolar do you know if this was an intentional choice?

**Comment by AdamRJensen** [comment]
> Would `atan2d` be a good addition to this PR too? Or [`arctan2`](https://numpy.org/doc/stable/reference/generated/numpy.arctan2.html), following last comment.

Certainly not a bad idea - I think I'll hold off adding it until I need it though :P

**Comment by kandersolar** [comment]
`asind` is among the oldest bits of pvlib python: https://github.com/pvlib/pvlib-python/commit/e74644ba99d16113fb976493b5006ffdf7fb1671#diff-0122268ff3ca7c61f9b9be534e547dbdfff3a7509577e704e456f06091908dfbR250-R252

I presume the choice of name was motivated by convenience when translating to python from MATLAB (which also uses [asind](https://www.mathworks.com/help/matlab/ref/asind.html)).

**Review by kandersolar — APPROVED** [review]
Since normally I probably wouldn't be in favor of adding functions to `pvlib.tools` without it being used somewhere in pvlib, I'll point out that we expect to use `atand` in an upcoming PR.

Thanks @AdamRJensen

**Pull request merged**